src/features/movies/movieSlice.js

```diff
@@ -1,42 +1,51 @@
 import { createEntityAdapter, createSlice } from "@reduxjs/toolkit";
-import {fetchAllMovies, fetchOneMovie} from "../imdb.service";
+import {fetchAllMovies, fetchOneMovie, fetchSearchMovies} from "../imdb.service";
 
 export const moviesAdapter = createEntityAdapter({
     selectId: (movie) => movie.id,
 })
+// export const searchesAdapter = createEntityAdapter({
+//     selectId: (movie) => movie.id,
+// })
 
 const movieSlice = createSlice({
     name: "movies",
     initialState: moviesAdapter.getInitialState({loading: false}),
-    reducers: {
-        cleanPreviousMovieDetails: (state)=>{
-            state.oneMovie = {};
-        }
-    },
+    reducers: {},
     extraReducers: {
         [fetchAllMovies.pending]: ()=>{
             console.log("Pending...");
         },
         [fetchAllMovies.fulfilled]: (state, {payload})=>{
             console.log("Fetched successfully :) ");
+            state.loading = true;

```

Comment:
As discussed in previous comments, loading shouldn't be handled here. I know redux-toolkit provides a way for this, but in my experience handling here isn't of much use.